### DISCUSSION TITLE
Give ideas and suggestions here!

### DISCUSSION THREAD

**Discussion by SpamixOfficial** [SpamixOfficial/OpenClick#1] [Ideas]
## 👋 Welcome!
  Here you can give ideas for updates and more!
  Remember to:
  * Be Nice!
  * Talk Positive!
  * Welcome others and be open-minded. Remember that this is a community we
  build together 💪.

**Comment by spynetS** [comment]
Add a delay to the hotkey so when you hold the hot key you can choose how many times a second it will click
